.circleci/config.yml

```diff
@@ -90,7 +90,7 @@ jobs:
           command: sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0-dev python-dev
       - run:
           name: Install Dependencies
-          command: npm install --quiet
+          command: sudo npm install -g yarn && npx lerna bootstrap
```

Comment:
Why don't you install `yarn` here in the `sudo apt-get install` command like you do above?

Reply:
I originally tried it that way: see [the first commit](https://github.com/mrice32/protocol/commit/e22cac1084cabb56e0119880cfe930e581f517ce). The [slither job](https://app.circleci.com/pipelines/github/UMAprotocol/protocol/7095/workflows/9a3d6815-fc52-4587-b85f-36a4f0744a56/jobs/51560/steps) broke with some strange yarn error (something peculiar to that docker image's environment I presume), so I ended up going with this method instead since it worked 🤷 .

Reply:
huh weird. Is it the same machine as the other CI sections?

Reply:
It's a different docker image: `trailofbits/eth-security-toolbox`. This is the only job that uses it.